Update active-directory-conditional-access-no-modern-authentication.md

MSFT-davidgregory opened the pull request:
The ADFS authorization rules you have listed here are all "permit" rules, which won't block anything until you actually remove the default Allow authorization rule from ADFS. So I prefaced both ADFS authorization rules section with the following:

For the following rules to effectively block legacy authentication, you first need to delete the default "Permit Access to All Users" authorization rule on ADFS.

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

MSFT-davidgregory committed:
Update active-directory-conditional-access-no-modern-authentication.md

The ADFS authorization rules you have listed here are all "permit" rules, which won't block anything until you actually remove the default Allow authorization rule from ADFS. So I prefaced both ADFS authorization rules section with the following:

For the following rules to effectively block legacy authentication, you first need to delete the default "Permit Access to All Users" authorization rule on ADFS.
